src/createServer.js

```diff
@@ -1,3 +1,56 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const text = normalizedURL.pathname.slice(1);
+    const toCase = normalizedURL.searchParams.get('toCase');
+    const errors = [];
+
+    if (!text) {
+      errors.push('Text to convert is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".');
```

Comment:
you can move this to variable
```suggestion
        + 'Correct request is: ${...}.');
```